src/main/java/modtrekt/storage/JsonAdaptedModule.java

```diff
@@ -81,7 +85,9 @@ public Module toModelType() throws IllegalValueException {
         }
         final ModTaskCount modelTaskCount = new ModTaskCount(taskCount);
 
-        Module created = new Module(modelCode, modelName, modelCredit, modelTaskCount);
+        final boolean modelIsDone = isDone;
+
+        Module created = new Module(modelCode, modelName, modelCredit, modelTaskCount, modelIsDone);
         return created;
```

Comment:
This can be inlined.
```suggestion
        return new Module(modelCode, modelName, modelCredit, modelTaskCount, modelIsDone);
```